dpl-go.json

```diff
@@ -0,0 +1 @@
y the CMS.","fields":[{"name":"value","description":"The raw text value.","args":[],"type":{"kind":"SCALAR","name":"String","ofType":null},"isDeprecated":false,"deprecationReason":null},{"name":"processed","description":"The processed text value.","args":[],"type":{"kind":"SCALAR","name":"Html","ofType":null},"isDeprecated":false,"deprecationReason":null},{"name":"format","description":"The text format used to process the text value.","args":[],"type":{"kind":"SCALAR","name":"String","ofType":null},"isDeprecated":false,"deprecationReason":null},{"name":"summary","description":"The processed text summary.","args":[],"type":{"kind":"SCALAR","name":"Html","ofType":null},"isDeprecated":false,"deprecationReason":null}],"inputFields":null,"interfaces":[],"enumValues":null,"possibleTypes":null},{"kind":"OBJECT","name":"Translation","description":"Available translations for content.","fields":[{"name":"title","description":"The title of the translation.","args":[],"type":{"kind":"SCALAR","name":"String","ofType":null},"isDeprecated":false,"deprecationReason":null},{"name":"langcode","description":"The language of the translation.","args":[],"type":{"kind":"NON_NULL","name":null,"ofType":{"kind":"OBJECT","name":"Language","ofType":null}},"isDeprecated":false,"deprecationReason":null},{"name":"path","description":"The path to the translated content.","args":[],"type":{"kind":"SCALAR","name":"String","ofType":null},"isDeprecated":false,"deprecationReason":null}],"inputFields":null,"interfaces":[],"enumValues":null,"possibleTypes":null},{"kind":"SCALAR","name":"UntypedStructuredData","description":"Untyped structured data. Eg JSON, configuration, settings, attributes.","fields":null,"inputFields":null,"interfaces":null,"enumValues":null,"possibleTypes":null}],"directives":[{"name":"skip","description":"Directs the executor to skip this field or fragment when the `if` argument is true.","locations":["FIELD","FRAGMENT_SPREAD","INLINE_FRAGMENT"],"args":[{"name":"if","description":"Skipped when true.","type":{"kind":"NON_NULL","name":null,"ofType":{"kind":"SCALAR","name":"Boolean","ofType":null}},"defaultValue":null}]},{"name":"include","description":"Directs the executor to include this field or fragment only when the `if` argument is true.","locations":["FIELD","FRAGMENT_SPREAD","INLINE_FRAGMENT"],"args":[{"name":"if","description":"Included when true.","type":{"kind":"NON_NULL","name":null,"ofType":{"kind":"SCALAR","name":"Boolean","ofType":null}},"defaultValue":null}]},{"name":"deprecated","description":"Marks an element of a GraphQL schema as no longer supported.","locations":["FIELD_DEFINITION","ENUM_VALUE"],"args":[{"name":"reason","description":"Explains why this element was deprecated, usually also including a suggestion for how to access supported similar data. Formatted using the Markdown syntax (as specified by [CommonMark](https://commonmark.org/).","type":{"kind":"SCALAR","name":"String","ofType":null},"defaultValue":"\"No longer supported\""}]}]}}
```

Comment:
This file is rather big. Is it recommended to commit it? Would be nice to not having it lying in the repository. Is it possible to generate it when we build the application?
I guess it is meant for being able to serve the schema quickly, without hitting code?

Reply:
Right now I don't think we are using the file at all actually. But I think you are right about it being used for serving the schema quickly. Maybe we should just omit the files for now and generate them if we need them in the future. We would also have to update the files everytime we make changes to the schema.